schema/v1.json

```diff
@@ -0,0 +1,113 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Amber SLSA buildType",
+  "description": "This schema describes the Amber SLSA buildType. It can be used to test whether a given SLSA predicate follows this buildType. It does not test compliance to the SLSA spec itself.",
+  "type": "object",
+  "properties": {
+    "predicateType": {
+      "const": "https://slsa.dev/provenance/v0.2"
+    },
+    "predicate": {
+      "type": "object",
+      "required": [
+        "buildType",
+        "buildConfig",
+        "materials"
+      ],
+      "properties": {
+        "buildType": {
+          "const": "https://github.com/project-oak/transparent-release/amber-slsa-buildtype/v1.json"
```

Comment:
This path is not valid. It should also be fixed in the example. 

Also, having a `v1.json` directly under `schema` does not look like a good file structure to me. I will soon add a schema for endorsements, which will also go somewhere in the `schema`  folder.  We need a better file structure. 

I can think of two options:

```
transparent-release
|-- schemas
|    |-- v_i
|    |    |-- amber_slsa_buildtype.json
|    |    |-- amber_endorsement.json
|    |    |-- some_example_file.json
|    |-- v_j
|-- ...
```

or 

```
transparent-release
|-- schemas
|    |-- amber_slsa_buildtype
|    |    |-- some_example_file.json
|    |    |-- v_i.json
|    |    |-- v_j.json
|    |-- amber_endorsement
|    |    |-- v_i.json
|-- ...
```

I don't have a strong preference between these two options, but the latter allows for the endorsement and the build-type to evolve separately, which might be what we want. The former, makes more sense if we look at the set of the schemas for the transparent-release as a whole that we'd publish and release together.

Reply:
Good point on both the link and the structure. I like the second one. I think the ability to update them separately is good to have at this point. :)